charts/mozmoderator/values.yaml

```diff
@@ -13,8 +13,6 @@ configMap:
     ANON_ALWAYS: true
     AWS_DEFAULT_REGION: us-west-2
     FROM_NOREPLY: Mozilla Moderator <no-reply@localhost>
-    MOZILLIANS_API_BASE: https://mozillians.org/api/v1/users/
```

Comment:
These variables are no longer used by Moderator. Moreover mozillians.org is decommissioned.

Reply:
great, thanks for this!

Reply:
going to get this in with failing CI then make another PR updating a few spots & tag you on it Tasos, so you can see a few places these variables get picked up.

Reply:
(i don't see a good way to propose changes on lines not yet touched but affected by these variables being removed)